quadtree.nim

```diff
@@ -1,41 +1,44 @@
-import fmath, sequtils
+import fmath
 
 const maxInQuadrant = 5
 
 type
-  ## The position and dimensions of a bounding box (usually, a Rect)
-  Bounds* = concept b
-    b.x is float32
-    b.y is float32
-    b.w is float32
-    b.h is float32
-
-  ## An element that can provide a bounding box
-  BoundsProvider* = concept q
-    boundingBox(q) is Bounds
-
   ## An element that can be stored in a quadtree.
   Quadable* = concept q
```

Comment:
Would it be good style to ditch the Quadable concept altogether, now that it can be specified with a `Rect` instead? I used the "old-style" experimental concepts here, and I'm not sure if it's worth making an entire concept just for the equality operator. 

[The standard library](https://github.com/nim-lang/Nim/blob/version-1-4/lib/system.nim#L1801) doesn't seem to use concepts (or any checks at all) even when a generic proc requires a `==` implementation.

Reply:
Concept is just a way of documentation, if you think it is cleaner to remove concept and instead add few words into comments why not.

Reply:
>The standard library doesn't seem to use concepts (or any checks at all) even when a generic proc requires a == implementation.

Std does not use them as they are quite new. No one bother to make pr with some axioms, thus we don't have any.